manim/mobject/graphing/functions.py

```diff
@@ -97,8 +97,8 @@ def construct(self):
 
     def __init__(
         self,
-        function: Callable[[float, float], float],
-        t_range: Sequence[float] | None = None,
+        function: Callable[[float], Sequence[float] | np.ndarray],
+        t_range: Sequence[float] | np.ndarray | None = None,
```

Comment:
```suggestion
        function: Callable[[float], Point3D],
        t_range: Sequence[float] | npt.NDArray | None = None,
```
For details on `Point3D` see here: https://docs.manim.community/en/latest/reference/manim.typing.html#manim.typing.Point3D

Is there a reason you chose not to change `t_range` to be `tuple[float, float]`?

Reply:
In the example code for Parametric Function:
```py
from manim import *

class PlotParametricFunction(Scene):
    def func(self, t):
        return np.array((np.sin(2 * t), np.sin(3 * t), 0))

    def construct(self):
        func = ParametricFunction(self.func, t_range = np.array([0, TAU]), fill_opacity=0).set_color(RED)
        self.add(func.scale(3))
```
t_range is a numpy array, so I wasn't sure whether we wanted to support that or just rely on plain tuples 😓

Reply:
Ah makes sense. Admittedly, I'm not a fan of the numpy array approach, since it could make users think that these *have* to be `NDArray`'s. What do you think of removing the `np.array` calls in the example code (wherever it makes sense to)? The typehint can still stay